packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -248,13 +248,13 @@ export async function* OpenAICompletionModel(
 
   for await (const event of openAiEventsToJson<CreateCompletionResponse>(responseIterator)) {
     logger.trace({ event }, 'Got createCompletion event');
+    yield event.choices[0].text;
     resultSoFar += event.choices[0].text;
-    yield resultSoFar;
   }
 
   logger.debug({ completion: resultSoFar }, 'Finished createCompletion');
```

Comment:
Hmm – is this log line redundant with the built-in render log for this component? If we remove this, we can also remove `resultsSoFar`, right?

Reply:
Nothing (currently anyway) automatically logs the result of rendering, so it's not redundant. (Perhaps they should be logged though.)